voxpupuli-acceptance.gemspec

```
@@ -14,7 +14,7 @@ Gem::Specification.new do |s|
 
   # Testing
   s.add_runtime_dependency 'bcrypt_pbkdf'
-  s.add_runtime_dependency 'beaker', '>= 4.2.0'
+  s.add_runtime_dependency 'beaker', '4.22.1'
```

I think you can also use a more free syntax, hoping 4.23.1 fixes it.
```suggestion
  s.add_runtime_dependency 'beaker', '>= 4.2.0', '!= 4.22.0', '!= 4.23.0'
```

See https://guides.rubygems.org/patterns/#declaring-dependencies